docs/dataset-overviews/anthropogenic-slcf-co2-emissions.md

```diff
@@ -1,83 +1,176 @@
-
-If you have any feedback, please add it to the [relevant GitHub discussion](https://github.com/PCMDI/input4MIPs_CVs/discussions).
-
-#### CMIP7 AR7 fast track
-
-No data available for this phase yet.
-
-This data will be for the CMIP7 AR7 fast track.
-All data sets for use in the fast track will be published with a `mip_era` metadata value of 'CMIP7'.
-This metadata value will appear both in the file's global metadata as well as its metadata on ESGF.
-
-Further details will follow in early 2025.
-
-#### CMIP7
-
-No data available for this phase yet.
-
-This data will be for CMIP7.
-All data sets for use in CMIP7 will be published with a `mip_era` metadata value of 'CMIP7'.
-This metadata value will appear both in the file's global metadata as well as its metadata on ESGF.
-
-Further details will follow after the fast track is underway
-(including details about how updates to this data will be handled over the lifetime of CMIP7).
-
-<!--- end-cmip7-phases-source-ids -->
-
-<!--- placeholder for piControl recommendation -->
-## Navigating the data
-
-### Recommendation for pre-industrial control
-
-Apply the 1850 values on repeat.
-
-<!--- end of placeholder for piControl recommendation -->
-
-<!--- begin-revision-history -->
-<!--- Do not edit this section, it is automatically updated when the docs are built -->
-<!--- No revisions, hence section is blank -->
-<!--- end-revision-history -->
+<!--- These values are used by `fill-out-auto-generated-sections.py` -->
+<!--- forcing="Anthropogenic short-lived climate forcer (SLCF) and CO<sub>2</sub> emissions" -->
+<!--- source_id_stub="CEDS-CMIP" -->
+# Anthropogenic short-lived climate forcer (SLCF) and CO<sub>2</sub> emissions
+
+## Key contacts
+
+- Names: Rachel Hoesly, Steve Smith
+- Emails: rachel.hoesly@pnnl.gov; ssmith@pnnl.gov
+
+## Summary
+
+CEDS v_2024_10_21 and CEDS v_2024_11_25 make up the CMIP6Plus anthropogenic short-lived climate forcer and CO<sub>2</sub> emissions data set including: 
+
+- Bulk emissions for all species except NMVOC's and supplementary solid biofuel emissions files from v_2024_11_25
+- Air emissions for all species, NMVOC bulk emissions, and supplementary speciated VOCs from v_2024_10_21
+
+These data releases are documented [here](https://zenodo.org/records/13952845) and [here](https://zenodo.org/records/14145000).
+
+This data is derived from CEDS v_2024_07_08 aggregate emissions release. CEDS v_2024_07_08 aggregate data by country and sector is also available in units of kilo-tonne (kt) per year which can be found [here](https://zenodo.org/records/12803197).
+
+Note that country totals in these summary files do not include international shipping or aircraft emissions, which are reported under the "global" iso. 
+
+Emissions data are provided for SO<sub>2</sub>, NOx, BC, OC, NH<sub>3</sub>, NMVOC,  CO, CO<sub>2</sub> from 1750 - 2022.
+Emissions data for CH<sub>4</sub> and N<sub>2</sub>O are provided from 1970 - 2022.

```

Comment:
```suggestion
CEDS data is available for testing (see the section below).
Emissions data are provided for SO<sub>2</sub>, NOx, BC, OC, NH<sub>3</sub>, NMVOC,  CO, CO<sub>2</sub> from 1750 - 2022.
Emissions data for CH<sub>4</sub> and N<sub>2</sub>O are provided from 1970 - 2022.

When handling this data, some extra care must be taken given that CEDS' testing data is published under multiple source IDs. In addition, solid biofuel emissions and speciated NMVOCs have the suffix "-supplemental" added to their source ID to avoid inadvertent double counting.

- Bulk emissions for all species except NMVOC's and supplementary solid biofuel emissions files are identified by the source ID "CEDS-CMIP-2024-11-25" (or "CEDS-CMIP-2024-11-25-supplemental" for the solid biofuel emissions)
- Air emissions for all species, NMVOC bulk emissions, and supplementary speciated VOCs are identified by the source ID "CEDS-CMIP-2024-10-21" (or "CEDS-CMIP-2024-10-21-supplemental" for the speciated VOCs)

The underlying data releases are documented [here](https://zenodo.org/records/13952845) and [here](https://zenodo.org/records/14145000).

This data is derived from CEDS' v_2024_07_08 aggregate emissions release. Aggregate data by country and sector for this release is also available in units of kilo-tonne (kt) per year and can be found [here](https://zenodo.org/records/12803197). Note that country totals in these summary files do not include international shipping or aircraft emissions, which are reported under the "global" iso. 

```

I would suggest doing this to avoid confusion and be clearer about what is for testing and what isn't. In the input4MIPs universe, we try and rely on source IDs so would suggest talking about those rather than versions as much as possible.